keyboards/keebio/rorschach/keymaps/tuesdayjohn/keymap.c

```diff
@@ -0,0 +1,296 @@
+ |        | Colemak| Qwerty |        |        |        |                    | Numpad |        |        |        |        |  RESET |
+ |--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+ |        |        |        |        |        |        |                    |        |NKRO Tog|        |        |        |        |
+ |--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+ |        |        |        |        |        |        |                    |        |        |        |        |        |        |
+ |--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+ |        |        |        |        |        |        |                    |        |        |        |        |        |        |
+ `--------------------------------------------+--------+--------.  ,--------+--------+--------------------------------------------'
+                                              |        |        |  |        |        |
+                                              `-----------------'  `-----------------'
+*/
+[_ADJUST] = LAYOUT(
+  _______, COLEMAK,  QWERTY, _______, _______, _______,                      NUMPAD,  _______, _______, _______, _______, RESET,
+  _______, _______, _______, _______, _______, _______,                      _______, NKROTG,  _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
+                                               _______, _______,    _______, _______
+),
+
+[_ADJUST2] = LAYOUT(
+  _______, COLEMAK,  QWERTY, _______, _______, _______,                      NUMPAD,  _______, _______, _______, _______, RESET,
+  _______, _______, _______, _______, _______, _______,                      _______, NKROTG,  _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,                      _______, _______, _______, _______, _______, _______,
+                                               _______, _______,    _______, _______
+)
+
+};
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _NUMBERS, _NUMBERS2, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case COLEMAK:
+      if (record->event.pressed) {
+        default_layer_set(1UL << _COLEMAK);
+        layer_move (_COLEMAK);
+
+      }
+      return false;
+      break;
```

Comment:
`break` after `return` is dead code.

Reply:
Honestly, I think that removing `return false` is better than removing break. 

Some functionality breaks if you return false (such as one shots).